src/amplitude-snippet.js

```diff
@@ -6,6 +6,11 @@
   as.src = 'https://d24n15hnbwhuhn.cloudfront.net/libs/amplitude-2.4.1-min.gz.js';
   var s = document.getElementsByTagName('script')[0];
   s.parentNode.insertBefore(as, s);
+  amplitude.Identify = function(){this.p={'a':{},'s':{},'u':{},'so':{}};};
```

Comment:
This is great. I think it would be better to make the list of proxied functions an array in the same manner as line 20 below, and programmatically generate these prototype functions (they all have very similar structure)